application.py

```diff
@@ -14,8 +14,7 @@
     ]
 }
 
-manager = Manager(application)
-manager.add_command("runserver", Server(port=5003))
+init_manager(application, 5003, ['./app/content/frameworks'])
```

Comment:
Should be `manager = init_manager(...)`, otherwise `manager.run()` won't work. But it looks like https://github.com/alphagov/digitalmarketplace-utils/blob/master/dmutils/flask_init.py#L80 doesn't return `manager` either, so we need to update utils first.